source/initial_temperature/interface.cc

```diff
@@ -121,6 +121,18 @@ namespace aspect
     }
 
 
+
+    template <int dim>
+    void
+    Manager<dim>::update()
+    {
+      // Go through the list, and update plugins:
```

Comment:
```suggestion
      // Go through the list and update plugins:
```
or just get rid of these comments.